packages/generate-persisted-query-manifest/cli.js

```diff
@@ -1,10 +1,46 @@
 #!/usr/bin/env node
 
-const { generatePersistedQueryManifest } = require("./dist/index.js");
+const { Command } = require("commander");
+const { cosmiconfig } = require("cosmiconfig");
+const { generatePersistedQueryManifest, defaults } = require("./dist/index.js");
+const { TypeScriptLoader } = require("cosmiconfig-typescript-loader");
```

Comment:
I can't say I get the point of letting you load TS files if there's no obvious way to teach the TS file what the type it's supposed to return is? Or are you able to import PersistedQueryManifestConfig within the file?

Reply:
Yep! You're able to import the type in the config file, which gives you autocompletion and inline docs for each option.

It looks like this (also in the PR description):

```ts
// persisted-query-manifest.config.ts
import { PersistedQueryManifestConfig } from '@apollo/generate-persisted-query-manifest';

const config: PersistedQueryManifestConfig = {
  documents: 'path/to/documents/**/*.{ts,tsx}'
};

export default config;
```